app/assets/javascripts/components/containers/mastodon.jsx

```diff
@@ -56,6 +56,7 @@ import uk from 'react-intl/locale-data/uk';
 import zh from 'react-intl/locale-data/zh';
 import bg from 'react-intl/locale-data/bg';
 import { localeData as zh_hk } from '../locales/zh-hk';
+import bg from 'react-intl/locale-data/ar';
```

Comment:
It seems that your commitment and other people 's commit are misleading. This is a slightly strange condition. You should make changes from the current master.